mediawiki/core/CVE-2018-0503.yaml

```diff
@@ -0,0 +1,17 @@
+title:     $wgRateLimits (rate limit / ping limiter) entry for 'user' overrides that for 'newbie'
+link:      https://phabricator.wikimedia.org/T169545
+reference: composer://mediawiki/core
+cve:       CVE-2018-0503
+branches:
+    1.27.x:
+        time: 2018-09-20 18:59:33
+        versions: ['>=1.27.0', '<1.27.5']
+    1.29.x:
+        time: 2018-09-20 19:24:54
+        versions: ['>=1.29.0', '<1.29.3']
```

Comment:
Was 1.28 safe ? If no, you either miss an affected range or should change that lower bound.

Reply:
Same as above, 1.28 was EOL at the time.